hcsshim.go

```diff
@@ -119,3 +162,17 @@ func convertAndFreeCoTaskMemString(buffer *uint16) string {
 	coTaskMemFree(unsafe.Pointer(buffer))
 	return str
 }
+
+func convertAndFreeCoTaskMemBytes(buffer *uint16) ([]byte, error) {
+	// TODO this requires an extra copy of the returned string data,
+	// it should be replaced with a more efficient way.
+	return []byte(convertAndFreeCoTaskMemString(buffer)), nil
```

Comment:
can we make one that doesn't make a string and then cast it? casting string to []byte allocates a new buffer.